.github/CODEOWNERS

```diff
@@ -4,55 +4,56 @@
+/plugins/catalog-graph                            @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/circleci                                 @backstage/reviewers @adamdmharvey
+/plugins/cloudbuild                               @backstage/reviewers @trivago/ebarrios
+/plugins/code-coverage                            @backstage/reviewers @alde @nissayeva
+/plugins/code-coverage-backend                    @backstage/reviewers @alde @nissayeva
+/plugins/cost-insights                            @backstage/reviewers @backstage/silver-lining
+/plugins/explore                                  @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/explore-react                            @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/fossa                                    @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/gcalendar                                @backstage/reviewers @szubster @ptychu @kielosz @alexrybch
+/plugins/git-release-manager                      @backstage/reviewers @erikengervall
+/plugins/home                                     @backstage/reviewers @backstage/techdocs-core
+/plugins/ilert                                    @backstage/reviewers @yacut
+/plugins/jenkins                                  @backstage/reviewers @timja
+/plugins/jenkins-backend                          @backstage/reviewers @timja
+/plugins/kafka                                    @backstage/reviewers @nirga
+/plugins/kafka-backend                            @backstage/reviewers @nirga
+/plugins/newrelic-dashboard                       @backstage/reviewers @mufaddal7
+/plugins/scaffolder-backend-module-rails          @backstage/reviewers @angeliski
+/plugins/scaffolder-backend-module-yeoman         @backstage/reviewers @pawelmitka
+/plugins/search                                   @backstage/reviewers @backstage/techdocs-core
+/plugins/search-*                                 @backstage/reviewers @backstage/techdocs-core
+/plugins/sonarqube                                @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/stack-overflow                           @backstage/reviewers @backstage/techdocs-core
+/plugins/stack-overflow-backend                   @backstage/reviewers @backstage/techdocs-core
+/plugins/techdocs                                 @backstage/reviewers @backstage/techdocs-core
+/plugins/techdocs-*                               @backstage/reviewers @backstage/techdocs-core
+/tech-insights-backend                            @backstage/reviewers @xantier @iain-b
+/tech-insights-backend-module-jsonfc              @backstage/reviewers @xantier @iain-b
+/tech-insights-tech-insights-common               @backstage/reviewers @xantier @iain-b
+/tech-insights-tech-insights-node                 @backstage/reviewers @xantier @iain-b
+yarn.lock                                         @backstage/reviewers @backstage-service
```

Comment:
Looks like a duplicate down here. Also not sure why this file is displayed as completely changed, is it possible to keep the formatting similar enough for it to show the local diff?